Cargo.toml

```diff
@@ -22,9 +22,11 @@ clap = "2.33.3"
 prost = "0.6"
 kelvin = "0.18"
 kelvin-radix = "0.10"
+futures = { version = "0.3", default-features = false, features = ["alloc"] }
 thiserror = "1.0"
 anyhow = "1.0"
 rustc_tools_util = "0.2"
+tower = "0.3"
```

Comment:
I heard somewhere tower is discontinued. Are we sure that isn't the case?

Reply:
It's needed by the tests.

See:
```
// Create the client binded to the default testing UDS path.
        let channel = Endpoint::try_from("http://[::]:50051")?
            .connect_with_connector(service_fn(|_: Uri| {
                // Connect to a Uds socket
                UnixStream::connect(SOCKET_PATH)
            }))
            .await?;
        let mut client = EchoerClient::new(channel);
```
Tower provides us this `service_fn(|_|...` which we need to setup the `Channel`. If you know any workarround, tell me.